tutorial/write-custom-widget.md

```diff
@@ -288,5 +290,8 @@ func main() {
 	w.ShowAndRun()
 }
 ```
+The widget should look like this:
+	
+![widgetDemoScreenSmall](https://user-images.githubusercontent.com/94919638/145826575-53ae12fc-e6d0-4976-9037-dbdaaea9f4de.png)
 
-If you stretch the widget the text will remain centered.
+	The widget can be resized and the text will remain centered.
```

Comment:
Is this indent intentional?

Reply:
> I have included a section of code to demonstrate how the widget works when the theme is changed however the code `theme.DarkTheme()` has been depreciated. I have looked for the equivalent without success. Usually a recommendation is given in the depreciation warning but I could not see it!

There is no recommended replacement because doing this is not recommended.
User settings should be respected and the `SetTheme` API is now focused on apps that provide a custom theme.